changelogs/fragments/1043-bug-title-zos_operator-is-passing-wrong-value-to-zoauopercmd.yml

```diff
@@ -0,0 +1,7 @@
+bugfixes:
+  - zos_operator - Corrected variable passed from wait_arg to wait in kwargs to zoau.
```

Comment:
Will ask to stick to this style when writing bugfixes changelogs "Use past tense to describe the problem and present tense to describe the fix." from (https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment).

We could say in the past tense that the module ignored the wait arg and fix now honors the wait time as implemented by zoau (?).

Reply:
I keep forgetting tense from the style guide.  Thanks for seeing that.  I'm pushing a fix shortly.

Reply:
Hey don't worry I do too, is always easier to remember when doing a code review